miasm2/jitter/emulatedsymbexec.py

```diff
@@ -90,11 +90,22 @@ def _simp_handle_segm(self, e_s, expr):
         """Handle 'segm' operation"""
         if expr.op != "segm":
             return expr
+
         segm_nb = int(expr.args[0].arg)
         segmaddr = self.cpu.get_segm_base(segm_nb)
-        return e_s(m2_expr.ExprOp("+",
-                                  m2_expr.ExprInt(segmaddr, expr.size),
-                                  expr.args[1]))
+
+        arch_size = self.ir_arch.attrib
```

Comment:
Using the `attrib` will only work for *x86*. For instance, on *arml*, the `attrib` value is `l`.
In [`llvmconvert`](https://github.com/cea-sec/miasm/blob/master/miasm2/jitter/llvmconvert.py#L738), I rather use the size of `PC`